test/functional/access-token.test.js

```diff
@@ -13,11 +13,25 @@ describe('oauth2 accessToken', () => {
 	});
 
 	it('should refresh accessToken with valid refreshToken', async () => {
+		const pdClient = lib.ApiClient.instance;
+		const oauth2 = pdClient.authentications.oauth2;
+
+		oauth2.host = 'http://localhost:1080';
+		oauth2.clientId = 'fakeClientId';
+		oauth2.clientSecret = 'fakeClientSecret';
+		oauth2.redirectUri = 'https://example.org';
+		oauth2.refreshToken = 'fakeRefreshToken';
```

Comment:
Most of this is repeating in each test. Could we move it to beforeEach?

Reply:
should be resolved :)